packages/calling/src/CallingClient/calling/callManager.ts

```diff
@@ -421,6 +429,21 @@ export class CallManager extends Eventing<CallEventTypes> implements ICallManage
   public getActiveCalls = (): Record<string, ICall> => {
     return this.callCollection;
   };
+
+  /**
+   * @param deviceId -.
```

Comment:
Avoid adding @param and @returns in the function definition as they are only for js file not ts. Just plain text describing the purpose of function should be enough.

Reply:
Removed @params.